src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Ethereum Energy Consumption
+description: The basic information you need to understand Ethereum's energy consumption.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum's Energy Consumption {#introduction-to-ethereum-energy-consumption}
+
+Since inception, sustainability has been a core value for Ethereum. However, until recently sustainability has sat uncomfortably alongside two other core values: security and scaleability. This page will explain why building out Ethereum to its current state has required a significant upfront carbon cost and why Ethereum is now on the cusp of becoming a truly low-carbon blockchain without undermining its other core principles.
```

Comment:
> Since inception, sustainability has been a core value for Ethereum.

I don't think we should be making generalised statements like this. PoS was always in the plan but calling sustainability a core value seems inaccurate.

```suggestion
Since inception, sustainability has been a core value for Ethereum. However, until recently sustainability has sat uncomfortably alongside two other core values: security and scalability. This page will explain why building out Ethereum to its current state has required a significant upfront carbon cost and why Ethereum is now on the cusp of becoming a truly low-carbon blockchain without undermining its other core principles.
```